services/businessconfig/src/main/java/org/opfab/businessconfig/services/ProcessesService.java

```diff
@@ -687,4 +680,15 @@ public void deleteAllBusinessData() throws IOException {
         File dataDirectory = new File(resourcePath.toString());
         FileUtils.cleanDirectory(dataDirectory);
     }
+
+    private void checkInputDoesNotContainsForbiddenCharacters(String inputName,String inputValue) throws ApiErrorException {
```

Comment:
Please correct the indentation in all the function.

Reply:
Done